working/macros/feature-specification.md

```diff
@@ -304,20 +304,69 @@ class, allowing you to build up your code fragments however you like.
 
 ## Scoping
 
-**TODO**: Fill in this section with more detail and a real proposal.
+### Resolved identifiers
 
 Macros will likely want to introduce references to identifiers that are not in
 the scope of the library in which they are running, but are in the scope of the
-macro itself.
+macro itself, or possibly even in some other library that is known to the
+macro, but not imported by the library where the macro is applied.
```

Comment:
"known to the macro" is vague. Instead of "in the scope of the macro itself or possibly...", I would just say: "in scope where the macro is defined and not where it is applied".